internal/messages/broadcast.go

```diff
@@ -21,18 +25,34 @@ type broadcastWrapper struct {
 	onDone func()
 }
 
+// We want to implement [memberlist.NamedBroadcast] because it goes through a
+// fast path when queueing messages for broadcasting, where the Name is
+// stored in a map to immediately invalidate older messages with the same
+// Name.
```

Comment:
Just curious again; this fast path is taken inside the memberlist code, and we don't need to retrurn memberlist.NamedBroadcast in ckit?

Reply:
Right, ckit internally implements the interface and memberlist hits the fast path, nothing for users to do here.